lms/djangoapps/instructor/views/api.py

```diff
@@ -1031,6 +1031,29 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
 
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+def get_students_who_may_enroll(request, course_id):
+    """
+    Initiate generation of a CSV file containing information about
+    students who may enroll in a course.
+
+    Responds with JSON
+        {"status": "... status message ..."}
+
+    """
+    course_key = CourseKey.from_string(course_id)
+    query_features = ['email']
+    try:
+        instructor_task.api.submit_calculate_may_enroll_csv(request, course_key, query_features)
+        success_status = _("Your students who may enroll report is being generated! You can view the status of the generation task in the 'Pending Instructor Tasks' section.")  # pylint: disable=line-too-long
```

Comment:
Don't disable pylint messages that can be reasonably resolved. You can break the string into sections, as Python autoconcatinates strings between parens, allowing you to break this into multiple lines.